### DISCUSSION TITLE
How admin plugin rights work? Or better - how they should do?

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#4650] [Q&A Core]
I am asking this because I need the answer how it should work and then I can say if this is bug or not.

- there is site updated for 2.3.1 
- I am not the first user (main admin by default)
- I am the main admin

![image](https://user-images.githubusercontent.com/5429548/148258145-9e9db875-2d76-4ac9-9937-290e9cf537ae.png)

- at that time (when adding main admin rights) , all rights above were checked and I had full access to site

- during time other main admin (with ID = 1) installed new plugins 
- I don't have access to them as main admin of site .  I think/know it is because this:

![image](https://user-images.githubusercontent.com/5429548/148258573-3ec7301d-eaa5-4ad9-b158-6c54567fdea4.png)

How should this work?  Why plugin access has priority before main admin access?
Or it is plugin related issue?
     
Thanks

**Comment by Moc** [comment]
I suspect this is a bug. @CaMer0n What do you think?
